document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/handler/CompanyReportDataHandler.java

```diff
@@ -309,6 +309,7 @@ private String toJson(CompanyReport companyReport, String companyNumber,
 
         try {
             LOG.infoContext(requestId, "Attempting to convert company report to JSON", getDebugMap(companyNumber));
+            LOG.info(" Attempting to convert company report to JSON companyReport :  " +companyNumber +": "+ companyReport);
```

Comment:
will this output anything meaningful without the class overriding a toString method?

Reply:
*CompanyReport class

Reply:
Yeah , I think should work. I was using it but then IDE was showing unused so removed